CMakeLists.txt

```diff
@@ -502,6 +503,36 @@ endif()
 # initialize CLI11 submodule
 cpp_cc_git_submodule(CLI11 BUILD PACKAGE CLI11 REQUIRED)
 
+# =============================================================================
+# Enable NMODL code-generator support
+# =============================================================================
+if(NRN_ENABLE_NMODL OR NRN_ENABLE_CORENEURON)
+
+  set(NMODL_ENABLE_PYTHON_BINDINGS
+      OFF
+      CACHE BOOL "Enable NMODL python bindings")
+  nrn_add_external_project(nmodl)
+  set(CORENRN_NMODL_BINARY ${CMAKE_BINARY_DIR}/bin/nmodl${CMAKE_EXECUTABLE_SUFFIX})
+  set(NMODL_TARGET_TO_DEPEND nmodl)
```

Comment:
This can be unconditionally replaced with `nmodl::nmodl` by now? (IIRC, there are only 2-3 places where this variable is used)